### PR TITLE
[wip] Feature/repliers from db

### PR DESCRIPTION
**Pull request by ronyb29**
progress so far:

- refactored file system dependencies out of repliers and commands to be able to change them with some DB Implementation in the near future. 
- also defined most of the repliers as "data based repliers", using the machinery that we'd use for replier configs coming form a DB.
- changed some of the tests to use fakes instead of mocks, since they're abstracted now.

### CODE REVIEW

File: giru/adapters_firebase.py [hunk 1]
```diff
@@ -0,0 +1,21 @@
+import time
```
Comment (ronyb29) [inline giru/adapters_firebase.py:1]: this file was just renamed just renamed

File: giru/adapters_fs.py [hunk 2]
```diff
@@ -0,0 +1,88 @@
+import csv
```
Comment (ronyb29) [inline giru/adapters_fs.py:1]: These implementation were just taken as is from the old `repliers.py` and `commands.py`

File: giru/core/commands/__init__.py [hunk 3]
```diff
@@ -0,0 +1,200 @@
+import datetime
```
Comment (ronyb29) [inline giru/core/commands/__init__.py:1]: moved from the old commands.py

**Comment by kengru** [comment]
Should I merge it? Asking because it says "wip" on the title still.

**Pull request merged**